test/SqrtPriceMath.t.sol

```diff
@@ -249,7 +249,7 @@ contract SqrtPriceMathTestTest is Test, GasSnapshot {
         assertEq(amount0, 0);
     }
 
-    function test_getAmount0Delta_returns0_1Amount1ForPriceOf1To1_21() public {
+    function test_getAmount0Delta_1Amount1ForPriceOf1To1_21() public {
```

Comment:
hm I know not your change but what is 1Amount1? Not really sure what this is testing/what the change to the name means

Reply:
It tests getAmount0Delta, with an amount1 of 1, and a price of SQRT_RATIO_121_100.
I removed the `return0` because it doesnt test that (i think it was copied from the test name above)

Reply:
sounds good